core/templates/pages/exploration-editor-page/feedback-tab/thread-table/thread-table.component.ts

```diff
@@ -12,39 +12,44 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import { Component, Input } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+import { SuggestionThread } from 'domain/suggestion/SuggestionThreadObjectFactory';
+import { DateTimeFormatService } from 'services/date-time-format.service';
+import { ThreadStatusDisplayService } from '../services/thread-status-display.service';
+
 /**
  * @fileoverview Directive for displaying the list of threads in the feedback
  * tab of the exploration editor.
  */
 
-require('filters/string-utility-filters/truncate.filter.ts');
+@Component({
+  selector: 'thread-table',
```

Comment:
Change the selector to `oppia-thread-table`.

Reply:
Done.